addons/crm/models/crm_team.py

```diff
@@ -78,3 +130,66 @@ def action_your_pipeline(self):
             ]
         action['context'] = action_context
         return action
+
+    def _compute_dashboard_button_name(self):
+        for team in self:
+            if team.use_opportunities:
+                team.dashboard_button_name = _("Pipeline")
+            else:
+                super(Team, team)._compute_dashboard_button_name()
```

Comment:
yeah, but I don't call it on self, I call it on team :p 
For loop just continues afterwards I believe.

Though it might be shorter indeed: you prefer somethings like this or is it less readable ?
```
def _compute_dashboard_button_name(self):
    opportunity_teams = self.filtered('use_opportunities')
    opportunity_teams.update({'dashboard_button_name': _("Pipeline")})
    super(Team, self-opportunity_teams)._compute_dashboard_button_name()
```